src/cpr_data_access/models/search.py

```diff
@@ -100,31 +113,6 @@ def sort_order_must_be_valid(cls, sort_order):
             )
         return sort_order
 
-    @field_validator("keyword_filters")
```

Comment:
It may not be for now but looking at some of the other field_validator methods we may be able to replace them with different field types. 

ie. `ids_must_fit_pattern` -> something like `family_ids: str = Field(regex=r"^[0-9]+.[0-9]+.[0-9]$")`

Then we also have a str not null validator as well.

Reply:
Refactoring these models is a good shout, I've actually already written up a ticket for revisiting these there is too much of a time crunch to do it now: https://linear.app/climate-policy-radar/issue/PDCT-965/speed-up-and-simplify-by-not-loading-pydantic-model-twice-per-query